$NetBSD: patch-configure,v 1.5 2017/06/28 15:35:48 he Exp $

Recognize sparc and powerpc linux (or BSD...) platforms.
Don't do doxygen.

--- configure.orig	2016-07-21 01:15:41.000000000 +0000
+++ configure
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/pkg/bin/bash
 ##
 ##  configure
 ##
@@ -113,6 +113,8 @@ all_platforms="${all_platforms} armv7s-d
 all_platforms="${all_platforms} armv8-linux-gcc"
 all_platforms="${all_platforms} mips32-linux-gcc"
 all_platforms="${all_platforms} mips64-linux-gcc"
+all_platforms="${all_platforms} powerpc-linux-gcc"
+all_platforms="${all_platforms} sparc-linux-gcc"
 all_platforms="${all_platforms} sparc-solaris-gcc"
 all_platforms="${all_platforms} x86-android-gcc"
 all_platforms="${all_platforms} x86-darwin8-gcc"
@@ -175,19 +177,6 @@ if [ "`cd \"${source_path}\" && pwd`" !=
   fi
 fi
 
-# check installed doxygen version
-doxy_version=$(doxygen --version 2>/dev/null)
-doxy_major=${doxy_version%%.*}
-if [ ${doxy_major:-0} -ge 1 ]; then
-    doxy_version=${doxy_version#*.}
-    doxy_minor=${doxy_version%%.*}
-    doxy_patch=${doxy_version##*.}
-
-    [ $doxy_major -gt 1 ] && enable_feature doxygen
-    [ $doxy_minor -gt 5 ] && enable_feature doxygen
-    [ $doxy_minor -eq 5 ] && [ $doxy_patch -ge 3 ] && enable_feature doxygen
-fi
-
 # disable codecs when their source directory does not exist
 [ -d "${source_path}/vp8" ] || disable_codec vp8
 [ -d "${source_path}/vp9" ] || disable_codec vp9
